scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/common/Scheduler.java

```diff
@@ -1302,8 +1302,7 @@ String getTaskServerLogsByTag(String id, String taskTag)
      * @throws UnknownJobException
      *             if one of the job IDs wasn't found
      */
-    List<JobInfo> getJobsInfoList(List<String> jobsId)
-            throws PermissionException, NotConnectedException, UnknownJobException;
+    List<JobInfo> getJobsInfoList(List<String> jobsId) throws PermissionException, NotConnectedException;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)